es6/experiment.js

```diff
@@ -100,7 +100,20 @@ class Experiment {
   }
 
   assign(params, args) {
-    throw "IMPLEMENT THIS";
+    throw "IMPLEMENT assign";
+  }
+
+  experimentParameters() {
+    throw "IMPLEMENT experimentParameters";
+  }
+
+  shouldFetchExperimentParameters(name) {
+    let experimentalParams = this.experimentParameters();
+    if (typeof experimentalParams == "boolean") {
```

Comment:
It seems that this.experimentParameters() should always be an array with positive length.